src/types.ts

```diff
@@ -688,15 +692,15 @@ export function responseType(
   methodDesc: MethodDescriptorProto,
   typeOptions: { keepValueType?: boolean; repeated?: boolean } = {}
```

Comment:
Looks like the `typeOptions` param is not used anymore? Lets just drop it if so?